demo-server.html

```diff
@@ -11,11 +11,39 @@
 <p>
 <a href="#introduction">Introduction</a><br />
 <a href="#insight">Using the OMERO.insight client</a><br />
-<a href="#webstart">Using the OMERO.insight Java Web Start client</a><br />
 <a href="#web">Using the OMERO.web client</a><br />
 </p>
 </div><!-- End Page Navigation -->
 
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="warning">
+<p class="warning-title">Terms of Use!</p>
+
+<p class="warning-text">By using the OMERO Demo Server, you agree to the following:</p>
+
+<ul>
+
+<li>You use this service at your own risk, and we provide NO GUARANTEE OR WARRANTY WHATSOEVER regarding uptime, security, data provenance or access, or any other service.</li>
+
+<li>You should not use the OMERO Demo Server as a repository for any important data, or data that must be held in any secure or speçific way.</li>
```

Comment:
Another typo carried over from pasting from Jason's text - the 'c' of 'specific' is a special character